test/simpleTestForGenerator/OPL-UK-3.0.txt

```diff
@@ -0,0 +1,113 @@
+United Kingdom Open Parliament Licence v3.0
+
+Open Parliament Licence
+
+You are encouraged to use and re-use the information that
+is available under this licence freely and flexibly, with
+only a few conditions. Using information under this licence
+
+Use of copyright and database right material made
+available under this licence (the information) indicates
```

Comment:
aha! I found the problem - (the information) needs to be (the 'information') to match!